src/App.tsx

```diff
@@ -26,9 +26,9 @@ const App = () => {
           {plantsCollection.map((item) => {
             return (
               <Route
-                key={item.plantID}
+                key={item.id}
```

Comment:
item is vague - maybe a rename to `plant`?

Reply:
renamed!